hedera-node/hedera-app/src/main/java/com/hedera/node/app/workflows/handle/record/SystemTransactions.java

```diff
@@ -420,7 +420,7 @@ public void dispatchNodeRewards(
         requireNonNull(now);
         requireNonNull(activeNodeIds);
         requireNonNull(nodeRewardsAccountId);
-        final var systemContext = newSystemContext(now, state, dispatch -> {});
+        final var systemContext = newSystemContext(now, state, dispatch -> {}, false);
```

Comment:
Just for clarity, this is change only on genesis and node rewards is because  the systemContext version is used only in these two places. For post upgrade set up we still depend on user transaction triggering the post upgrade transactions.